public/sketch.js

```diff
@@ -27,11 +29,14 @@ function stringToColor(str) {
 
 // Create new user
 function createNewUser(id) {
-  users[id] = {
+  users[id] = { 
     username: '',
     pressed: [],
     color: stringToColor(id)
   }
+  //create a independent osc as create new user
+  setTone(id);
+  console.log(osc);
```

Comment:
Generally, you want to remove these before you publish your code. Not a big deal right now, just trying to teach you guys slowly. =P